modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -75,8 +109,17 @@ definition {
 				locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
 		}
 
-		task ("And When Save As the custom view") {
-			FDSCustomView.createNewView(key_nameView = "TEST");
```

Comment:
My feeling is that this modification is unneeded. The scope of the task is to automate the test CanBeEdited. So any other modifications to other tests are suspicious to me.